tests/conftest.py

```diff
@@ -7817,3 +7820,13 @@ def finalizer():
 
     request.addfinalizer(finalizer)
     return factory
+
+
+@pytest.fixture(scope="session")
+def virtual_host_style_acess_for_rgw_session():
+    """
+    Session scoped fixture to configure Virtual host style access for RGW
+
```

Comment:
I would add the BZ which is blocking this testing to doc of the fixture:
[2283643](https://bugzilla.redhat.com/show_bug.cgi?id=2283643)

Reply:
Updated in [ad92b0c](https://github.com/red-hat-storage/ocs-ci/pull/9839/commits/ad92b0cff453642274d40cfe7a212c93dba1aaab).